backend/app/models/experiment_template.py

```diff
@@ -37,10 +37,15 @@ class ExperimentTemplate(Document):
     class Settings:
         name = "experimentTemplates"
 
-    def initialize_files(self, dockerfile, pip_requirements, script):
+    def initialize_files(self, base_image, pip_requirements, script):
         base_path = settings.get_experiment_template_path(template_id=self.id)
         base_path.mkdir(exist_ok=True, parents=True)
 
+        with open("app/data/template-Dockerfile") as f:
```

Comment:
I recommend using some standard templating options such as jinja2. For example follow the usage as done in [BentoML](https://docs.bentoml.org/en/latest/guides/containerization.html#dockerfile-template)